backend/spec/models/collaborator_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe Collaborator, type: :model do
+  let :attrs do 
+    {
+      numberOfCTLEmployees: '4',
+      numberOfPJColaborators: '5',
+      numberOfInterns: '6',
+      last_update: 10.seconds.ago
+    }
+end
+
+  context 'with validation problems' do
+    it 'field not a number' do
+      %i[numberOfCTLEmployees numberOfPJColaborators numberOfInterns].each do |k|
+        attrs[k] = 'ten'
+      expect(described_class.new(attrs)).to be_invalid
+      end
+    end
```

Comment:
Cuidado com a semântica aqui. O que você fez testa se é inválido _caso todos os atributos da lista não sejam números_. Acho que você queria testar se é inválido caso _cada um não seja número_ certo? Se for isso, coloca o laço por fora do `it "..."`